### PR TITLE
Feature/104 refbox

### PR DESCRIPTION
**Pull request by satanja**

### CODE REVIEW

File: front/src/RefboxField.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  console.log(
+    "Button " + props.tag + " was pressed from team " + props.teamColor
+  );
+};
+
+/**
+ * Places the buttons in 2 columns and rows of 3
+ * Allows for multiple refboxes to be next to each other (float : left)
+ */
+const RefboxStyle = styled.div`
```
Comment (g868) [inline front/src/RefboxField.js]: Think this should be `Refbox`

File: front/src/RefboxField.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+/**
+ * Constructs a refbox for a team consisting of 6 buttons:
+ * - Kick off (KO)
+ * - Free kick (FK)
+ * - Goal kick (GK)
+ * - Throw in (TI)
+ * - Corner (C)
+ * - Penalty (P)
+ * Also takes care of handeling commands that the buttons represents
```
Comment (helaan) [inline front/src/RefboxField.js]: handling

File: front/src/RefboxField.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    </div>
+  );
+};
+
+/**
+ * The function that handles button presses
+ */
+const handleCommand = props => {
+  console.log(
+    "Button " + props.tag + " was pressed from team " + props.teamColor
```
Comment (helaan) [inline front/src/RefboxField.js]: Why not pass the command pressed to the parent via a function prop?

File: front/src/RefboxButton.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+};
+
+/**
+ * A styled component defining the style of the refbox buttons
+ */
+const Button = styled.button`
+  background-color: ${props => props.teamColor};
+  width: 20.6rem;
+  height: 20.6rem;
+  border: 0.25rem solid;
```
Comment (g868) [inline front/src/RefboxButton.js]: Do we make the border width dependent on the window size?

File: front/src/RefboxButton.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+/**
+ * A button part of the refbox. Does a callback to its parent to handle onClick events
+ * Author: S.A. Tanja
+ * Author: G.W. van der Heijden
+ *
+ * Props:
+ *  - teamColor: the team color of the of button (cyan or magenta).
```
Comment (g868) [inline front/src/RefboxButton.js]: "of the of"?

File: front/src/RefboxField.js (hunk 2 above)
Comment (g868) [inline front/src/RefboxField.js]: *represent

File: front/src/RefboxField.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+};
+
+/**
+ * Places the buttons in 2 columns and rows of 3
+ * Allows for multiple refboxes to be next to each other (float : left)
+ */
+const RefboxStyle = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 20.6rem);
+  grid-template-rows: repeat(3, 20.6rem);
```
Comment (g868) [inline front/src/RefboxField.js]: 20,6 rem is used multiple times, maybe we should make a constant for this

File: front/src/RefboxButton.js (hunk 4 above)
Comment (gwvdh) [reply front/src/RefboxButton.js]: No

File: front/src/RefboxButton.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+import PropTypes from "prop-types";
+
+/**
+ * A button part of the refbox. Does a callback to its parent to handle onClick events
+ * Author: S.A. Tanja
+ * Author: G.W. van der Heijden
+ *
+ * Props:
+ *  - teamColor: the team color of the button (cyan or magenta).
+ *  - tag: the tag of the button according to the TRC.
```
[the remark this answers is not in the document]
Comment (gwvdh) [reply front/src/RefboxButton.js]: Fixed

**Pull request closed without merging**